src/Serialization/upgrades/0.11.2.jl

```diff
@@ -0,0 +1,88 @@
+################################################################################
+# Upgrade Summary
+# Defines an OscarBasictype, and any Vector{OscarBasictype} will not have
+# backrefs in any entry. Instead all entries of the array will have their
+# representative only and the entry_type is given for all entries once on
+# the save level as the :vector key
+
+upgrade_script = UpgradeScript(
+    v"0.11.2",
```

Comment:
I don't think this comment was resolved. One way to resolve it would be to add a comment saying:
```suggestion
    v"0.11.2", # data format version to which this upgrade script applies
```
assuming this is the correct, that is (I don't know, I'd have to re-read the code to find out...)